cq_editor/widgets/editor.py

```diff
@@ -2,6 +2,7 @@
 from PyQt5.QtCore import pyqtSignal, QFileSystemWatcher, QTimer
 from PyQt5.QtWidgets import QAction, QFileDialog
 from PyQt5.QtGui import QFontDatabase
+from pathlib import Path
```

Comment:
Could you change to `from path import Path`?

Reply:
Sorry, forgot I did that on this PR as well!